python/paddle/fluid/tests/performance_tuning/DeepASR/run_on_1card.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Given that run_on_1card.sh is almost exactly the same as run_on_4cards.sh, it seems reasonable to merge them into one file with a command line parameter.